src/argilla/client/feedback/metrics/utils.py

```diff
@@ -0,0 +1,81 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from collections import defaultdict
+from typing import TYPE_CHECKING, Dict, List, Tuple, Union
+
+from argilla.client.feedback.schemas import RankingQuestion
+from argilla.client.feedback.schemas.enums import ResponseStatusFilter
+
+if TYPE_CHECKING:
+    from argilla.client.feedback.dataset import FeedbackDataset
+    from argilla.client.feedback.dataset.remote.dataset import RemoteFeedbackDataset
+
+
+# Type aliases
+Responses = List[Union[float, int, str]]
+Suggestions = Responses
+
+
+def get_responses_per_user(
```

Comment:
also, I might say this is a more generic utils and might belong in the module that @sdiazlor  is creating